### ISSUE TITLE
how to make the model perform better?

### ISSUE THREAD

**Issue by cs-heibao** [bourdakos1/Custom-Object-Detection#27]
@bourdakos1 
hi,I've trained with my own dataset, and the result is just okay, so I wonder which aspect I can do some changes to finetune the model, such as: the optimizier, learning rate or other parameters, and how to change ?
thanks !

**Comment by bourdakos1** [comment]
Hi @junjieAI, this repo relies on a dated version of the TensorFlow api. We've moved to a more future proof version here: https://github.com/cloud-annotations/training
I encourage you to try it out and reopen this issue there if you are still running into problems
